### DISCUSSION TITLE
venus-os_dbus-mqtt-dcdc ?

### DISCUSSION THREAD

**Discussion by ak68-hub** [mr-manuel/venus-os_dbus-mqtt-battery#22] [Ideas]
Hi Manuel,

I would lite to intergrate my Victron DC-DC Booster as well (after battery and solar).
I´ve got all datas in my MQTT-Broker.

Is it possible, to integrate it in the same way by DBUS-MQTT-DCDC - Script ?
Unfortunately I can´t find a script by Github :(

The DBUS-Variables are:

![image](https://github.com/mr-manuel/venus-os_dbus-mqtt-battery/assets/63199646/1020ff32-d8c9-404a-93f3-5b1767c89ead)

The adresses are (new) includes by Victron:

![image](https://github.com/mr-manuel/venus-os_dbus-mqtt-battery/assets/63199646/08fd0d65-c47d-4f07-85a5-cc7818659b62)

Thank you very much
Andreas

**Comment by mr-manuel** [comment]
Write me on Discord and we will arange something.
